woocommerce/woocommerce-android
# Display bundle products info

⚠️  This PR depends on this [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2709)
⚠️ Still missing unit tests

Closes: #8402

### Why
We are adding Product Bundles read-only support to the Woo App.

### Description
This PR adds support to see bundled products in the product details screen. To achieve this goal, we are observing the changes in the new BundledProducts table, which contains the bundled product information. Bundled products don't contain information like the product image or the product's SKU, so before displaying the bundled information, we retrieve this data from the product table. 

### Testing instructions
1. Open the products tab
2. Select a bundle product
3. Check that the bundled products card is shown
4. Tap on the bundled products card
5. Check that the bundled products detail screen is shown


### Images/gif

https://user-images.githubusercontent.com/18119390/231362040-5d743ac2-6eac-4020-a9b7-1c74e354b72f.mp4
